### ISSUE TITLE
Change clip tool background transparency or disable it entirely 

### ISSUE THREAD

**Issue by Samo99** [arch1t3cht/Aegisub#82]
Hi there,

Can you please make changes to clip tool so that we can either change the transparency to the background or disable the background, cause when you use the clip tool in dark scenes it makes it even darker, and it becomes hard to follow the lines correctly.

Ex: 

Scene before using clip tool
![1](https://github.com/arch1t3cht/Aegisub/assets/12115578/ce8a7f87-e081-45d2-aa55-65baea064d1b)

Scene when using the clip tool
![2](https://github.com/arch1t3cht/Aegisub/assets/12115578/617530e4-3432-4fc8-8516-4eedde0e5aa4)

**Comment by arch1t3cht** [comment]
There's an option to do just that under Options > Interface > Colors.
